src/cms/models/users/organization.py

```diff
@@ -16,11 +16,14 @@ class Organization(models.Model):
         verbose_name=_("slug"),
         help_text=_("Unique string identifier without spaces and special characters."),
     )
-    icon = models.ImageField(
-        null=True,
-        blank=True,
-        upload_to="organizations/%Y/%m/%d",
+
+    icon = models.ForeignKey(
+        "cms.MediaFile",
         verbose_name=_("logo"),
+        on_delete=models.SET_NULL,
+        related_name="icon_org",
```

Comment:
```suggestion
        related_name="organization_icon_of",
```
Just to stay consistent with the other models, I'd choose this related name.
I get that's not an ideal name, but if we change the naming scheme, we should probably change it for all models with an icon (regions, organizations, pages, events and POIs). Maybe `organizations_with_this_file_as_icon` would be most understandable, but sure that's a bit verbose :sweat_smile: 
Maybe, for the sake of simplicity, just `organizations`? if you go with this approach, please also rename the related name in all other models as well. The exact name is less relevant than the consistency :smile:

Reply:
Okay, yeah, I get this one! I might go with ```modelname_icon```, if that's fine.

There is another small thing, which came to my mind: In the organization_list_view the logo is represented as a link to the image source. Would it make sense to either represent the actual logo as an image instead of the link or just not show it as a table property?

Reply:
> Okay, yeah, I get this one! I might go with `modelname_icon`, if that's fine.

I think this sounds as if the attribute would return an icon instance instead of the model. What about `icon_modelname_plural`, so e.g. `icon_organizations` (similar to what you initially suggested)?

> There is another small thing, which came to my mind: In the organization_list_view the logo is represented as a link to the image source. Would it make sense to either represent the actual logo as an image instead of the link or just not show it as a table property?

Yes, I really like this idea. A small thumbnail image would be cool! :heart_eyes_cat:

Reply:
> I think this sounds as if the attribute would return an icon instance instead of the model. What about `icon_modelname_plural`, so e.g. `icon_organizations` (similar to what you initially suggested)?

That's fine to me as well, I'll go with it!